models/public/openpose-cmu-tf/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  This is TensoFlow version of OpenPose algorithm, used for multi-person
+  human pose estimation. The model is based on VGG pretrained network,
+  which described in the original paper <https://arxiv.org/pdf/1812.08008.pdf>.
+  For details see repository <https://github.com/ildoonet/tf-pose-estimation>.
+task_type: human_pose_estimation
+files:
+  - name: graph_opt.pb
+    size: 209299198
+    sha256: 1fc81fff68d2351dba0bf206ae4431a760a202151ff539feaefdaab534d79aeb
+    source: http://download686.mediafire.com/xk527xyl3r6g/qlzzr20mpocnpa3/graph_opt.pb
```

Comment:
This link no longer works (well, it does, but it doesn't directly point to the file anymore).

There is clearly no way to get a permanent download link from MediaFire. We could, in principle, add a dedicated source type for MediaFire, like we do for Google Drive, and try to screen scrape the download link from the MediaFire webpage, but I don't think we should. There is no way to do this reliably, as MediaFire could alter its design at any moment, or even put up a CAPTCHA.

I think we'll just have to write this model off as a lost cause.

Reply:
Yes, there is no direct link to the model. There is also no respond from authors on our request to place model somewhere else. @vladimir-dudnik, are we going to redistribute this model through another storage (download01.org?) or exclude it?